projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -134,10 +137,17 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
         )
         upper_cased_env = {k.upper(): v for k, v in os.environ.items()}
         for key in config_keys:
-            normalized_key = VDK_ + key.replace(".", "_").upper()
+            normalized_key_with_vdk_prefix = VDK_ + self.__normalize_key(key)
+            normalized_key = self.__normalize_key(key)
             value = upper_cased_env.get(normalized_key)
-            if value is not None:
-                log.debug(f"Found environment variable for key {key}")
```

Comment:
Maybe we can add a log message at the info level with a statistic of what environment variables were found and used to make it easier for the users to understand what happened, in case they end up in a situation, where they have environment variables for other tools with the same names as the ones we use?

Reply:
I think already our logs are way too verbose. For somebody new it's hard to get around them. I think we need to minimize INFO logs to the essential. Also I think we need to have them more structured (but that's separate topic)

But the problem you are pointing out (easier for the users to understand what happened and how something was configured) is real. 

But I am thinking of perhaps extending vdk config-help to show the current value and how it was configured.
Maybe printing stats may not be too much logging.  I'll merge this now and think about this problem in separate PR. Thanks for pointing it out.